add_to_app/flutter_module/.android/Flutter/build.gradle

```diff
@@ -0,0 +1,45 @@
+// Generated file. Do not edit.
```

Comment:
Hey @Swayam221 !  I think you have pushed a few other files other than the one which actually had the typo.

Reply:
> Hey @Swayam221 ! I think you have pushed a few other files other than the one which actually had the typo.

Hey @arteevraina ! Thankyou so much for pointing out the mistake. Shall I close this PR and open another for the same after fixing the PR?

Reply:
> Hey @Swayam221 ! I think you have pushed a few other files other than the one which actually had the typo.

Hey @arteevraina ! Thankyou for pointing out the mistake. I'll close this PR and open another for the same.